modules/api/pkg/ee/clusterbackup/backup/handler.go

```diff
@@ -113,6 +124,14 @@ func CreateEndpoint(ctx context.Context, request interface{}, userInfoGetter pro
 	}, nil
 }
 
+func getLabels(backupOrigin, projectID, clusterID string) map[string]string {
```

Comment:
Perhaps we should start them `kubermatic/` to clarify that these are added by KKP. Also, can we please create some constants for these keys instead of hardcoded string values?